nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java

```diff
@@ -262,8 +269,7 @@ private void printExecutionResults(Set<String> allFailures, CleanSurefireExecuti
 
     private void writeCurrentRunInfo(CleanSurefireExecution execution) {
```

Comment:
Since the function name signifies the info to be stored for the current run, why do we need to store all the info in the first run file?

Reply:
This is how the implementation works currently and there some dependent integration tests mentioned in the description. We are not modifying this approach but just avoiding the use of get(0).